R/process_set_values_to.R

```diff
@@ -70,20 +70,14 @@ process_set_values_to <- function(dataset,
   tryCatch(
     result <- mutate(dataset, !!!set_values_to),
     error = function(cnd) {
-      abort(
-        paste0(
-          "Assigning variables failed!\n",
-          "set_values_to = (\n",
-          paste(
-            " ",
-            names(set_values_to),
-            "=",
-            set_values_to,
-            collapse = "\n"
-          ),
-          "\n)\nError message:\n  ",
-          cnd
-        )
+      cli_abort(
```

Comment:
Should we specify the `class` argument and add the `call` argument to `process_set_values_to()` and pass it to `cli_abort()`?
The error message
```
#> Error in value[[3L]](cond) : Assigning variables failed!
```
looks a bit strange.

Reply:
Good catch. Updated!

<img width="714" alt="image" src="https://github.com/pharmaverse/admiraldev/assets/26774684/b903c1b1-82c7-46b1-a6f9-91dff48e628a">